ajax/libs/coolprop/6.3.0/coolprop.min.js

```diff
@@ -0,0 +1 @@
)}var l={type:e,opts:i,mountpoint:t,mounts:[]},u=e.mount(l);return u.mount=l,l.root=u,r?FS.root=u:n&&(n.mounted=l,n.mount&&n.mount.mounts.push(l)),u},unmount:function(e){var i=FS.lookupPath(e,{follow_mount:!1});if(!FS.isMountpoint(i.node))throw new FS.ErrnoError(ERRNO_CODES.EINVAL);var t=i.node,n=t.mounted,r=FS.getMounts(n);Object.keys(FS.nameTable).forEach(function(e){for(var i=FS.nameTable[e];i;){var t=i.name_next;-1!==r.indexOf(i.mount)&&FS.destroyNode(i),i=t}}),t.mounted=null;var a=t.mount.mounts.indexOf(n);assert(-1!==a),t.mount.mounts.splice(a,1)},lookup:function(e,i){return e.node_ops.lookup(e,i)},mknod:function(e,i,t){var n=FS.lookupPath(e,{parent:!0}).node,r=PATH.basename(e);if(!r||"."===r||".."===r)throw new FS.ErrnoError(ERRNO_CODES.EINVAL);var a=FS.mayCreate(n,r);if(a)throw new FS.ErrnoError(a);if(!n.node_ops.mknod)throw new FS.ErrnoError(ERRNO_CODES.EPERM);return n.node_ops.mknod(n,r,i,t)},create:function(e,i){return i=void 0!==i?i:438,i&=4095,i|=32768,FS.mknod(e,i,0)},mkdir:function(e,i){return i=void 0!==i?i:511,i&=1023,i|=16384,FS.mknod(e,i,0)},mkdirTree:function(e,i){for(var t=e.split("/"),n="",r=0;r<t.length;++r)if(t[r]){n+="/"+t[r];try{FS.mkdir(n,i)}catch(e){if(e.errno!=ERRNO_CODES.EEXIST)throw e}}},mkdev:function(e,i,t){return void 0===t&&(t=i,i=438),i|=8192,FS.mknod(e,i,t)},symlink:function(e,i){if(!PATH.resolve(e))throw new FS.ErrnoError(ERRNO_CODES.ENOENT);var t=FS.lookupPath(i,{parent:!0}).node;if(!t)throw new FS.ErrnoError(ERRNO_CODES.ENOENT);var n=PATH.basename(i),r=FS.mayCreate(t,n);if(r)throw new FS.ErrnoError(r);if(!t.node_ops.symlink)throw new FS.ErrnoError(ERRNO_CODES.EPERM);return t.node_ops.symlink(t,n,e)},rename:function(e,i){var t,n,r=PATH.dirname(e),a=PATH.dirname(i),o=PATH.basename(e),l=PATH.basename(i);try{t=FS.lookupPath(e,{parent:!0}).node,n=FS.lookupPath(i,{parent:!0}).node}catch(e){throw new FS.ErrnoError(ERRNO_CODES.EBUSY)}if(!t||!n)throw new FS.ErrnoError(ERRNO_CODES.ENOENT);if(t.mount!==n.mount)throw new FS.ErrnoError(ERRNO_CODES.EXDEV);var u,d=FS.lookupNode(t,o),s=PATH.relative(e,a);if("."!==s.charAt(0))throw new FS.ErrnoError(ERRNO_CODES.EINVAL);if("."!==(s=PATH.relative(i,r)).charAt(0))throw new FS.ErrnoError(ERRNO_CODES.ENOTEMPTY);try{u=FS.lookupNode(n,l)}catch(e){}if(d!==u){var c=FS.isDir(d.mode),_=FS.mayDelete(t,o,c);if(_)throw new FS.ErrnoError(_);if(_=u?FS.mayDelete(n,l,c):FS.mayCreate(n,l))throw new FS.ErrnoError(_);if(!t.node_ops.rename)throw new FS.ErrnoError(ERRNO_CODES.EPERM);if(FS.isMountpoint(d)||u&&FS.isMountpoint(u))throw new FS.ErrnoError(ERRNO_CODES.EBUSY);if(n!==t&&(_=FS.nodePermissions(t,"w")))throw new FS.ErrnoError(_);try{FS.trackingDelegate.willMovePath&&FS.trackingDelegate.willMovePath(e,i)}catch(t){console.log("FS.trackingDelegate['willMovePath']('"+e+"', '"+i+"') threw an exception: "+t.message)}FS.hashRemoveNode(d);try{t.node_ops.rename(d,n,l)}catch(e){throw e}finally{FS.hashAddNode(d)}try{FS.trackingDelegate.onMovePath&&FS.trackingDelegate.onMovePath(e,i)}catch(t){console.l
```

Comment:
This file does not appear in the latest tagged release, 6.3.0: https://github.com/CoolProp/CoolProp/tree/v6.3.0/wrappers/Javascript